contracts/core/Dispatcher.sol

```diff
@@ -555,7 +555,18 @@ contract Dispatcher is OwnableUpgradeable, UUPSUpgradeable, IDispatcher {
         // Only call if we are sure receiver is a contract
         // Note: This tx won't revert if the low-level call fails, see
         // https://docs.soliditylang.org/en/latest/cheatsheet.html#members-of-address
+        uint256 gasBeforeCall = gasleft();
         (success, message) = receiver.call(args);
+
+        // x is the amount of gas left right before the low level call. Solidity will allocate x*63/64 to the low level
+        // call
+        // and x/64 for the remaining execution after the low-level call. If this low-level call runs out of gas, then
+        // gasLeft will be equal to x/64 at the start of the remaining execution. so we should check gasBefore < 1/64
+        if (!success && gasleft() <= gasBeforeCall / 64) {
```

Comment:
While being a very small issue, it should be `gasBeforeCall / 63` not `gasBeforeCall / 64`.

See arguments in our implementation: https://github.com/catalystdao/GeneralisedIncentives/blob/f71a3ba9a50fa1ec3eec8b3409aa98ec625a199b/src/IncentivizedMessageEscrow.sol#L586